docs/test/test.md

```diff
@@ -100,7 +100,7 @@ The following steps are also needed to make sure libnetwork package could be fou
 ```
 BUILDPATH=/tmp/pouchbuild
 mkdir -p $BUILDPATH/src/github.com/docker
-cp -r /go/src/github.com/alibaba/pouch/extra/libnetwork $BUILDPATH/src/github.com/docker/libnetwork
+cp -r /go/src/github.com/alibaba/pouch/vendor/github.com/docker/libnetwork $BUILDPATH/src/github.com/docker/libnetwork
```

Comment:
No need to add libnetwork directory into GOPATH, it will vendor to build. I think just keep `$BUILDPATH` in `GOPATH`.

Reply:
OK. I was not sure about this.